understanding/status-messages.html

```diff
@@ -55,317 +55,216 @@
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 4.1.3:</span>Status Messages (Level AA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 4.1.3:</span>ステータスメッセージ (レベル AA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>In content implemented using markup languages, <a href="#dfn-status-message">status messages</a> can be <a href="#dfn-programmatically-determined">programmatically determined</a> through <a href="#dfn-role">role</a> or properties such that they can be presented to the user by <a href="#dfn-assistive-technology">assistive technologies</a> without receiving focus.
+                  <p>マークアップ言語を使って実装されたコンテンツでは、<a href="#dfn-status-message">ステータスメッセージ</a>は、<a href="#dfn-role">役割 (role) </a> 又はプロパティを通して<a href="#dfn-programmatically-determined">プログラムによる解釈が可能</a>であり、フォーカスを受けとらなくても<a href="#dfn-assistive-technology">支援技術</a>によって利用者に提示することができる。
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   			
-                  <dt>Goal</dt>
-                  <dd>Make users aware of important changes in content.</dd>
+                  <dt>目標</dt>
+                  <dd>利用者に重要なコンテンツの変化を知らせる。</dd>
                   			
-                  <dt>What to do</dt>
-                  <dd>Let assistive technology notify users about status changes that don't take focus.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>フォーカスを受け取らないステータスの変更について、支援技術が利用者に通知させる。</dd>
```

Comment:
「支援技術が利用者に通知させる。」という表現が若干わかりにくく感じました。「支援技術に、利用者への通知を、させる」という意味かと思います。若干意訳ですが、以下、いかがでしょうか。

```suggestion
                  <dd>フォーカスを受け取らないステータスの変更について、支援技術が利用者に通知できるようにする。</dd>
```

Reply:
単に「通知できる」としてみました。